### DISCUSSION TITLE
翻译后只出现一个释义

### DISCUSSION THREAD

**Discussion by Yoimiya42** [YiiGuxing/TranslationPlugin#5689] [Q&A]
不管是用有道还是百度，选中一个词翻译后，只出现一个中文释义，啥情况
![}O SQ1QVCQEL875)WF H1)H](https://github.com/YiiGuxing/TranslationPlugin/assets/165260565/0d6cb02d-9446-4265-8467-a25d572226a0)

**Comment by YiiGuxing** [comment]
有道原本是有一些额外的信息的，但现在已经被取消掉了，不再返回，原因不明，后面是否会恢复也不明。不过即将发布的新版本的微软翻译会带有词典和例句的。
